website/community/Working-Groups.mdx

```diff
@@ -0,0 +1,88 @@
+
+All the Contributors for the Documentation Working Group are invited to a bi-weekly sync call to discuss the completed, ongoing and pending work. If you would like to join as a volunteer, please fill-up the Google form: [**https://forms.gle/ibH2XsaBSAeiyQcX7**](https://forms.gle/ibH2XsaBSAeiyQcX7)
+
+Join the `#documentation` channel for further updates on the working group.
+
+### DevOps working group
+
+The DevOps working group is primarily aimed at a collaboration-first approach to software by facilitating transparency, shared workloads, continuous feedback and reviews, and robust documentation. As a part of the DevOps working group, we work on:
+
+-   Building CI/CD pipelines for various moja global projects to facilitate continuous testing, deployment and release lifecycle.    
+-   Integrating infrastructure-as-a-code tools to manage and deploy our infrastructure over the cloud.  
+-   Develop workflows and processes to help moja global teams capitalize on cloud services and resources for continuous development.    
+-   Implement observability across application metrics while assisting development teams with operational support.
+   
+As part of the DevOps working group, you work with the development teams and the documentation team to facilitate operations. Some of our current areas of focus are:
+
+-   **Build pipelines**: Develop a robust and flexible build for a wide array of moja global projects, like FLINT, Reporting Tool, to manage continuous integration and releases.  
+-   **Test suite**: Writing unit, integration and end-to-end test suite to be integrated into our core projects, like FLINT-UI, FLINT-Cloud, to help development teams to ship software confidently.    
+-   **Automation toolings**: Implementing automated toolings to help contributors with easy installation and setup to facilitate contributions.    
+-   **Container Images**: Centralizing container images scattered across the organization and projects under a single umbrella while implementing CI/CD to test and deploy images.
+ 
+We use a wide array of tool-sets and technologies like [Docker](https://docker.io/), [GitHub Actions](https://github.com/features/actions), [Terraform](https://www.terraform.io/), [BASH](https://www.gnu.org/software/bash/), [Amazon Web Services](https://aws.amazon.com/), [Google Cloud Platform](https://cloud.google.com/), [Kubernetes](https://kubernetes.io/), [Swagger](https://swagger.io/) to name a few.
+
+All the Contributors for the DevOps Working Group are invited to a bi-weekly sync call to discuss the completed, ongoing and pending work. If you would like to join as a volunteer, please join the `#devops` channel for further updates on the working group.
```

Comment:
All the contributors -> All contributors

Reply:
We don't have a #devops channel yet, but let's make one! Please clarify that you mean join on Slack.

Reply:
I have created one yesterday. We can start conversation on them!